### DISCUSSION TITLE
How to raise reject by logic use ui.upload?

### DISCUSSION THREAD

**Discussion by italanchan** [zauberzeug/nicegui#1620] [Q&A]
Checked ui.upload doc, there is on_rejected prop but don't know how to raise reject event by custom logic, e.g. 

- file size is too small (Min: 10Kb).
- the sheet 'keyword' not found in uploaded excel file.

**Comment by falkoschindler** [comment]
What exactly are you trying to achieve?
If you want to call a certain function for invalid uploads, e.g. for notifying the user, you don't need to use `on_rejected`, but can simply call it in your `on_upload` event handler. There you have access to the file size and content and can react accordingly.
